### DISCUSSION TITLE
How can I get the list of notes that contain specific tag?

### DISCUSSION THREAD

**Discussion by naive231** [darlal/obsidian-switcher-plus#98] [Q&A] [answered]
I know I can just using the built-in "Tags: show tags" to get the list, but I wondered that if switcher-plus can do that?

**Comment by darlal** [comment, marked as answer]
Hey @naive231 Switcher++ currently doesn't support filtering on tags. There are two other plugins that already implement this  [obsidian-better-command-palette](https://github.com/AlexBieg/obsidian-better-command-palette#file-searching-using-tags) and [obsidian-another-quick-switcher](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher). All 3 of these plugins can be installed and used simultaneously to mix and match the capabilities that you want.

**Comment by techeddie** [comment]
hi @naive231, here is another tools which i prefer to use. It shows you the notes (results) on the on search pane:
https://github.com/rwblickhan/obsidian-tag-search
